bridge/src/index.ts

```diff
@@ -296,6 +297,22 @@ process.on("uncaughtException", console.error);
         "string"
     );
 
+    const PLANET_ODIN_ID: string | undefined = Configuration.get(
+        "PLANET_ODIN_ID",
+        true,
+        "string"
+    );
+    const PLANET_HEIMDAL_ID: string | undefined = Configuration.get(
+        "PLANET_HEIMDAL_ID",
+        true,
+        "string"
+    );
+    const HEIMDALL_VALUT_ADDRESS: string | undefined = Configuration.get(
```

Comment:
no vault address for odin?

Reply:
I didn't add odin's valult address because it would not be needed.